Doc/library/configparser.rst

```diff
@@ -139,12 +139,20 @@ case-insensitive and stored in lowercase [1]_.
 Supported Datatypes
 -------------------
 
-Config parsers do not guess datatypes of values in configuration files, always
-storing them internally as strings.  This means that if you need other
-datatypes, you should convert on your own:
+Config parsers do not guess datatypes in configuration files, always storing
```

Comment:
IMO it would be better to separate the description regarding section names from the rest. I think it is far less important, and should therefore appear later on this page. The example should be separated and included with such a new paragraph, too.